assets/src/edit-story/components/library/panes/text/textSets/test/textSet.js

```diff
@@ -258,10 +258,11 @@ describe('TextSets', () => {
 
   it('should allow inserting a text set', () => {
     insertTextSet.mockImplementation((elements) => elements);
-    const { queryAllByRole } = setup(SETS);
+    const { queryAllByRole } = setup(SET);
     const sets = queryAllByRole('listitem');
-    expect(sets).toHaveLength(2);
-    fireEvent.click(sets[1]);
+    expect(sets).toHaveLength(1);
```

Comment:
Note: This test was faulty before -- there was a duplicate `listitem` per each text set and the test indeed found 2 listitems even though there is only one text set defined.